### ISSUE TITLE
upon startup: SyntaxWarning: invalid escape sequence '\w'

### ISSUE THREAD

**Issue by klauspdoll** [molpro/iMolpro#187]
version 5c2f086051472  yet non-reproducible

after startup:

```
iMolpro/molpro_input.py:586: SyntaxWarning: invalid escape sequence '\w'
  if line.strip() and line.strip()[0] != '{' and not re.match('^ *\w+ *=', line) and not in_group:

```

I have seen it twice, but cannot reproduce it now.

**Comment by pjknowles** [comment]
I can't reproduce this. It would be enough to provide an input that causes it. But I suspect that  it is fixed by making the pattern a raw string.

**Comment by klauspdoll** [comment]
I tried all possible inputs, to no avail.

Then I did 

git pull

and just started iMolpro     (no project opened, only the splash screen !!!):

`python iMolpro.py &`
(nothing else done!)
```
[1] 5548
molpro_input.py:591: SyntaxWarning: invalid escape sequence '\w'
  if line.strip() and line.strip()[0] != '{' and not re.match('^ *\w+ *=', line) and not in_group:

```

**Comment by pjknowles** [comment]
Yes, I agree, and I thought it was fixed in #189 .
